mycli/main.py

```diff
@@ -747,6 +749,13 @@ def format_output(self, title, cur, headers, status, expanded=False,
 
         return output
 
+    def get_reserved_space(self):
+        """Get the number of lines to reserve for the completion menu."""
+        reserved_space_ratio = .2
+        max_reserved_space = 8
+        _, height = click.get_terminal_size()
+        return min(int(height * reserved_space_ratio), max_reserved_space)
```

Comment:
At 4 or less, it comes back as 0. At 9 or less it comes back as 1, but that is then given to the toolbar. So, right now the completion window goes away at less than 10 lines. I guess that's a little harsh :)

I think I set it that way out of personal preference. For me, if I have a window that is 9 lines or less, I'd like to have at most, 1 line for completions (probably even zero).

But, I can see the argument for trying to keep completions present as much as possible. Let me try .4 and see how it looks.

At 4 lines or less (where 1 goes to the toolbar), we probably shouldn't have completions at all. Would you agree?

In master, mycli currently ceases to function below 9 lines.